### DISCUSSION TITLE
ISO install fails on bare metal installation (Installer will not load)

### DISCUSSION THREAD

**Discussion by negus6** [Security-Onion-Solutions/securityonion#2959] [Unsupported Versions]
Booting to CentosOS Linux 7 Core is successful, but the installation hangs at starting installer. The server was running SO 16.04 previously. As a troubleshooting measure I deleted all the partitions from the disk using fdisk to blow away the old installation... then tried installing SO 2. I also tried booting UEFE and Legacy using both DVD and USB installation media (verified iso). Still no luck.

**Comment by negus6** [comment]
Not sure what was going on with anaconda on the ISO with my hardware... but as a work around I was able to install Centos 7 separately and then do a network install of SO 2.

**Comment by Acewiza** [comment]
The problem with "bare metal" is there's alot of different kinds of it.

**Reply by negus6** [reply]
So true! And yet the Centos installer has yet to fail me, on any bare metal installation.

**Comment by TOoSmOotH** [comment]
Are you using NVME?

**Comment by Acewiza** [comment]
No, not on the onion.

On Mon, Feb 22, 2021 at 6:07 PM Mike Reeves <notifications@github.com>
wrote:

> Are you using NVME?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/2959#discussioncomment-395955>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASJEPQSWDRCGJ4QYBLUM3BDTAL5WFANCNFSM4XRM6BFQ>
> .
>


-- 
Paul D. Shaffer, MSgt, USAF (ret.)

**Reply by Acewiza** [reply]
Does syslog say anything about it?

**Comment by launchdaemon** [comment]
I also found that I was not able to boot from the ISO.  I tried using Etcher, `dd` and Rufus to create USB sticks, ensured that secure boot was disabled and tried with both UEFI and legacy boot enabled but could not start from it.

I was able to boot fine from an Ubuntu ISO `dd`'d to the same USB though.
